tasks/main.yml

```diff
@@ -5,7 +5,16 @@
   become: yes
   become_user: '{{ rvm1_user }}'
 
+- name: Check that RVM is installed
```

Comment:
I'm not too sure about this but I think its ok to let the role fail if nothing has been installed, rather than adding tasks like this to satisfy check mode. Do you mind adding a comment in the code here to explain this task to help avoid confusion (since the previous tasks are there to ensure RVM is installed)?

Reply:
I agree with you. For the moment, I made this to be "always" check mode compatible, but I am not sure if it is worth it. Up to core maintainers to decide...